aspnetcore/tutorials/first-web-api.md

```diff
@@ -453,6 +453,37 @@ Use Postman to delete a to-do item:
 * Set the URI of the object to delete (for example `https://localhost:5001/api/TodoItems/1`).
 * Select **Send**.
 
+<a name="over-post"></a>
+
+## Prevent over-posting
+
+Currently the app exposes the entire `TodoItem`. Productions apps typically limit the data that's input and returned using subset of the model. The subset model is called a Data Transfer Object (DTO), input model, or view model.
```

Comment:
What about trying to rephrase this first paragraph as follows? Feel free to dismiss this suggestion, if you don't like it.
```suggestion
Currently our sample app exposes the entire `TodoItem` object. Productions apps typically limit the data that's input and returned using subset of the model. There are multiple reasons behind this and security aspect is a major one. That subset of a model is usually referred to as a Data Transfer Object (DTO), input model, or view model. We will use the `DTO` naming in this article.
```